tests.sh

```diff
@@ -242,28 +244,45 @@ run_ior(){
     no_of_ps=$(($DAOS_CLIENTS * $PPC))
     echo
 
-    ior_cmd="ior
-             -a DFS -b $BLOCK_SIZE -w -W -r -R -g -G 27 -C -Q 1 -i 2 -s 1 -o /testFile 
-             -t $XFER_SIZE --dfs.cont $CONT_UUID
-             --daos.group daos_server --dfs.pool $POOL_UUID --dfs.oclass SX
-             --dfs.svcl $POOL_SVC -vv"
-
-    mpich_cmd="mpirun
+    IOR_WR_CMD="ior
+             -a DFS -b ${BLOCK_SIZE} -C -e -w -W -g -G 27 -k -i 2
+             -s ${SEGMENTS} -o /testFile
+             -O stoneWallingWearOut=1
+             -O stoneWallingStatusFile=${OUTPUT_DIR}/sw.${SLURM_JOB_ID} -D 60
+             -d 5 -t ${XFER_SIZE} --dfs.cont ${CONT_UUID}
+             --daos.group daos_server --dfs.pool ${POOL_UUID} --dfs.oclass SX
+             --dfs.svcl ${POOL_SVC} -vvv"
+
+    IOR_RD_CMD="ior
+             -a DFS -b ${BLOCK_SIZE} -C -Q 1 -e -r -R -g -G 27 -k -i 2
+             -s ${SEGMENTS} -o /testFile
+             -O stoneWallingStatusFile=${OUTPUT_DIR}/sw.${SLURM_JOB_ID}
+             -d 5 -t ${XFER_SIZE} --dfs.cont ${CONT_UUID}
+             --daos.group daos_server --dfs.pool ${POOL_UUID} --dfs.oclass SX
+             --dfs.svcl ${POOL_SVC} -vvv"
+
+    prefix_mpich="mpirun
              -np $no_of_ps -map-by node
              -hostfile Log/$SLURM_JOB_ID/daos_client_hostlist
-	     $ior_cmd"
+             $ior_cmd"
```

Comment:
ior_cmd not defined and should be removed?

Reply:
yes, you are right